pkg/options/rawyamltmpl/raw_text_applier.go

```diff
@@ -28,17 +28,20 @@ import (
 	bundle "github.com/GoogleCloudPlatform/k8s-cluster-bundle/pkg/apis/bundle/v1alpha1"
 	"github.com/GoogleCloudPlatform/k8s-cluster-bundle/pkg/converter"
 	"github.com/GoogleCloudPlatform/k8s-cluster-bundle/pkg/filter"
-	"github.com/GoogleCloudPlatform/k8s-cluster-bundle/pkg/maker"
+	"github.com/GoogleCloudPlatform/k8s-cluster-bundle/pkg/options"
 )
 
-// Maker makes components via go-templating, for objects stored as raw strings
+// applier applies options via go-templating, for objects stored as raw strings
 // in config maps.
-type Maker struct{}
+type applier struct{}
 
-// MakeComponent makes a component, implementing the make-component interface.
-// The raw text maker's MakeComponent method only uses inlined raw text as
-// input to the object-creation process.
-func (m *Maker) MakeComponent(comp *bundle.ComponentPackage, pm maker.ParamMaker, of *filter.Options) (*bundle.ComponentPackage, error) {
+// NewApplier creates a new optionts applier instance.
+func NewApplier() options.Applier {
+	return &applier{}
+}
+
+// ApplyOptions applies options to to raw-text that's been inlined.
+func (m *applier) ApplyOptions(comp *bundle.ComponentPackage, pm options.OptionsFn) (*bundle.ComponentPackage, error) {
```

Comment:
Use `common.ApplyCommon` (`common.ApplyOptions`, if you take my comment above)?

Reply:
It can't because of the way the raw text templates work -- they're filegroups.

Reply:
https://github.com/GoogleCloudPlatform/k8s-cluster-bundle/blob/master/pkg/apis/bundle/v1alpha1/component_types.go#L117

has some more details

Reply:
Please add inline comments to that affect, with more details about why you can't use `common.ApplyCommon`.

Reply:
Added some comments